index.js

```diff
@@ -4,15 +4,42 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "blue";
 
 
```

Comment:
Awesome! I don't see the first deliverable on your sheet, but you would first find the header element and save it to a variable and then change it! But you were correct :)